iwf/command.go

```diff
@@ -52,12 +50,12 @@ func NewInternalChannelCommand(commandId, channelName string) Command {
 	}
 }
 
-func NewTimerCommand(commandId string, firingTime time.Time) Command {
```

Comment:
Similar in Java using Duration, The standard way in Golang is using time.Duration. 

Also this is a breaking change here, not a huge problem to me. But we should make it clear in release -- (that's why I thought haivng a sparate PR makes it easier, but it's okay)

Reply:
Actually it may be better to keep compabitility for this api. And create a `NewTimerCommandByDuration` API.

Reply:
How would the TimerCommand.DurationSeconds be populated using this `firingTime time.Time` param?
`int64(firingTime.Sub(time.Now()).Seconds())`? And then adding a check if negative, set to 0?

Reply:
Yes 👍 that’s the idea (same in Java SDK, we used to convert duration to timestamp)

Reply:
We probably should panic if setting a negative timestamp

Reply:
This could lead to false panics though right? E.g.
firingTime = now + 500ms
... blah ... (this ends up taking 600ms)
NewTimer(firingTime) {  now -100ms  }

Reply:
yeah, good catch. Let's keep it as it then. 
Probably mark this method as "deprecated" now, in favor of NewTimerCommandByDuration -- It's more useful in practice and people should migrate to it.

Reply:
The reasons that I want to panic is that when this happened in the past, it's usally user workflow code issues, but it's a bit hard to figure out why (they used a static/constant for the timerCommand, whcih got the converted value forever during after deployed).

Reply:
What you described is possible, but It should be extremely rare that someone want to sleep for milliseconds. The temporal timer is not accurate at millisecond either. But it's possible that a state api call can take a few seconds, but that's also pretty rare.

I am okay to remove the panic, you can decide it.